### ISSUE TITLE
Add animated camera border when object is detected on the selected camera

### ISSUE THREAD

**Issue by brpeterso** [dermotduffy/frigate-hass-card#1252]
When viewing HA and looking at the video, you sometimes need to look very close to see if a person is in the video (especially on a phone). It would awesome if there was a way to animate the border (in and out of a selected color) to let you know if the current video feed detects an object based on cam config. You could also possibly have an icon show up as well, but I think the animated border would be cleaner. If this already exists in some way, can someone guide me?  Thanks!

**Comment by dermotduffy** [comment]
Scan Mode seems similar to what you want, although perhaps not identical. It will automatically select the camera that has detected occupancy, or motion, or entity of your choosing -- and will animate a border around the card when this occurs. 

[Screenshot](https://github.com/dermotduffy/frigate-hass-card#scan-mode)
[Configuration](https://github.com/dermotduffy/frigate-hass-card#view-scan-mode-configuration)

Have a play around and tell me if this is what you're looking for? I think the difference is that it is designed for when the card is not being interacted with -- then it'll automatically update the card to an "interesting" camera (e.g. for use on wall-mounted tablets that spend most of their time not being interacted with). I think you are looking for something more permanent that doesn't otherwise interfere with the usage of the card, just shows if there's something there or not.

Anyway, play around with scan mode and let me know what you think.

**Comment by brpeterso** [comment]
Thanks for the quick reply.  I have looked into scan mode, but as you mentioned I'm more interested in looking at one main camera that's focused on my driveway/entry sidewalk/street.  It's something that I have up on my HA main phone screen all the time and I was thinking it would be cool to give a visual if you're looking directly at it or if you see it in your peripheral vision.  I also struggled w/ the camera changing on me if I was waiting for someone to visit and someone from my house triggered another camera.  I thought I could maybe do this with card_mod and one of the frigate person sensors, but then I realized it may be difficult to figure out what camera is showing to figure out what sensor to use... unless I'm missing something.
